docs/RNCamera.md

```diff
@@ -599,7 +599,7 @@ writeExif = {
 
 - `orientation` (string or number). Specifies the orientation that us used for taking the picture. Possible values: `"portrait"`, `"portraitUpsideDown"`, `"landscapeLeft"` or `"landscapeRight"`.
 
-- `path` (file path on disk). Specifies the path on disk to save picture to.
+- `path` (file path on disk). Specifies the path on disk to save picture to. Picture taking process might be slower as picture is saved other than cache directory.
```

Comment:
Is it?

Do we have a benchmark for this?

Reply:
I did check on by seeing difference in System.time between starting of taking photo process to completion of photo taking process. And, cache directory was about 100ms faster.